src/utils/bootstrap/bootstrap.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@nestjs/common';
+import { EcoverseService } from 'src/domain/ecoverse/ecoverse.service';
+import { UserInput } from 'src/domain/user/user.dto';
+import { UserService } from 'src/domain/user/user.service';
+
+@Injectable()
+export class BootstrapService {
+  constructor(
+    private ecoverseService: EcoverseService,
+    private userService: UserService
+  ) {}
+
+  async boostrapEcoverse() {
+    try {
+      this.ecoverseService.getEcoverse();
+      const ctAdminDto = new UserInput();
+      ctAdminDto.name = 'ctAdmin';
+      ctAdminDto.email = 'admin@cherrytwist.org';
+      ctAdminDto.lastName = 'admin';
+
+      if (this.userService.getUserByEmail('admin@cherrytwist.org')) {
```

Comment:
Afraid logic does not look correct...if the user already exists you still want the person to be assigned as an admin